src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -1290,7 +1290,7 @@ protected unsafe void WriteEventWithRelatedActivityIdCore(int eventId, Guid* rel
                         // So we need to prevent this from getting written directly to the Listeners.
                         if (this.GetType() != typeof(NativeRuntimeEventSource))
 #endif // MONO && !TARGET_BROWSER
-                            WriteToAllListeners(eventId, pActivityId, relatedActivityId, eventDataCount, data);
+                        WriteToAllListeners(eventId, pActivityId, relatedActivityId, eventDataCount, data);
```

Comment:
While I don't approve of the existing code (I believe we should always use braces), we shouldn't be making it even harder to read.

```suggestion
                            WriteToAllListeners(eventId, pActivityId, relatedActivityId, eventDataCount, data);
```